client/src/lib/usbstick.test.ts

```diff
@@ -0,0 +1,74 @@
+import fakeKiosk from '../../test/helpers/fakeKiosk'
+import {
+  isAvailable,
+  isPresent,
+  isMounted,
+  doMount,
+  doUnmount,
+} from './usbstick'
```

Comment:
Why is there USB stuff in this PR?

Reply:
as per comment: because I needed to add some coverage and this was something that needed it.